x-pack/plugin/ql/build.gradle

```diff
@@ -8,10 +8,16 @@ esplugin {
   extendedPlugins = ['x-pack-core']
 }
 
+ext {
+  // EQL dependency versions
+  antlrVersion = "4.5.3"
+}
+
 archivesBaseName = 'x-pack-ql'
 
 dependencies {
   compileOnly project(path: xpackModule('core'))
+  compileOnly "org.antlr:antlr4-runtime:${antlrVersion}"
```

Comment:
This seems to cause issues downstream in that EQL and SQL projects do not bundle the antlr runtime in their packaging despite still being declared as an API.